scout/commands/base.py

```diff
@@ -52,20 +52,20 @@ def get_app(ctx):
     options = ctx.find_root()
     cli_config = {}
     # if a .yaml config file was provided use its params to intiate the app
-    if options.params["config"]:
+    if options.params.get("config"):
         with open(options.params["config"], 'r') as in_handle:
             cli_config = yaml.load(in_handle, Loader=yaml.FullLoader)
 
-    if options.params["demo"]:
+    if options.params.get("demo"):
         cli_config['demo'] = 'scout-demo'
 
     app = create_app(config=dict(
         DEBUG=True,
```

Comment:
Hm, should we really do that? But, anyways, hasn't changed.. :)

Reply:
Should do what?

Reply:
DEBUG=True in --demo? Often enough the case, I guess.. :)

Reply:
The app is initiated with that log level but after it is created the log of the cli is set to what the user indicates in the option. See the body of the 'cli' function. 
I'm testing that this works in a test named test_base_command.py